AlembicImporter/Assets/UnitTests/AlembicExportTest.cs

```diff
@@ -160,11 +147,28 @@ public class AlembicExportTest : AlembicTestBase {
             Assert.That (go, Is.Not.Null);
         }
 
-        // Loads a given scene
-        IEnumerator SceneLoader (string sceneToLoad) {
-            SceneManagement.EditorSceneManager.LoadScene (sceneToLoad, UnityEngine.SceneManagement.LoadSceneMode.Single);
-            yield return null;
+        public string GetAssetsAbsolutePath (string relPath) {
+            return Application.dataPath + relPath.Replace ("Assets", "");
         }
+
+    }
+
+    public class AlembicExportTest : AlembicTestBase {
+
+        private AlembicExporter GetAlembicExporter () {
+            var alembicExporter = Object.FindObjectOfType<AlembicExporter> ();
+            Assert.That (alembicExporter, Is.Not.Null);
+            return alembicExporter;
+        }
+
+
+
+        /// <summary>
+        /// Test that the abc file was reimported and can be loaded into Unity through
+        /// the asset database.
+        /// </summary>
+        /// <param name="abcPath"></param>
```

Comment:
the function moved but it's description stayed here :P. Please move the description too.